cbrain_task/fsl_melodic/portal/fsl_melodic.rb

```diff
@@ -80,22 +173,62 @@ def after_form #:nodoc:
   end
   
   def final_task_list #:nodoc:
-    
+    functional_ids = []
```

Comment:
Can be re-write this way: 
functional_ids =  params[:functional_file_ids].values # this return an array with all the value.